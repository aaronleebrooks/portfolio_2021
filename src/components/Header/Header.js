import Button from "../Button";

function Header() {
  return (
    <div>
      <span>Aaron Brooks</span>
      <Button>
        Resume
      </Button>
      <Button>
        Demos
      </Button>
      <Button>
        Contact Me
      </Button>
    </div>
  );
}

export default Header;
