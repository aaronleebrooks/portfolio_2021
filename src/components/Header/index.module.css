.headerBar {
    display: flex;
    justify-content: space-between;
}

.navSection {
    display: flex;
}

.headerText {
    font-family: Inter,sans-serif;
    font-weight: 700;
    letter-spacing: -0.5px;
    line-height: 1.42;
    color: rgb(22, 28, 45);
}

.largeHeaderText {
    font-size: 1.5rem;
}

.smallHeaderText {
    font-size: 15px;
}
