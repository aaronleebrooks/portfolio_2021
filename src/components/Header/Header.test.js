import { render, screen } from '@testing-library/react';
import Header from './Header';

describe('Header', () => {
  test('renders', () => {
    const {container} = render(<Header />);
    expect(container).toMatchSnapshot();
  });
})