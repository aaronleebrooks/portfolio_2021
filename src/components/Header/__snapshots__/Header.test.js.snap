// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders 1`] = `
<div>
  <div>
    <span>
      Aaron Brooks
    </span>
    <button>
      Resume
    </button>
    <button>
      Demos
    </button>
    <button>
      Contact Me
    </button>
  </div>
</div>
`;
